homeassistant/components/shelly/__init__.py

```diff
@@ -39,6 +39,7 @@
 @singleton.singleton("shelly_coap")
 async def get_coap_context(hass):
     """Get CoAP context to be used in all Shelly devices."""
+    # pylint: disable=no-member
```

Comment:
Why is this disabled ? Doesn't seem right. Are you having the latest aioshelly installed ?

Reply:
```suggestion
```